v1/types/types.gunk

```diff
@@ -698,3 +698,30 @@ const (
 	// RARE is rare status of access card.
 	RARE
 )
+
+// Address hold all details about Address
+type Address struct {
+	// CountryName holds the country name information.
+	CountryName string `pb:"1" json:"country_name"`
+
+	// CityName holds the city name information.
+	CityName string `pb:"2" json:"city_name"`
+
+	// State holds the state information.
+	State string `pb:"3" json:"state"`
+
+	// Street holds the street information.
+	Street string `pb:"4" json:"line_1"`
+
+	// PostalCode holds the postal code information.
+	PostalCode string `pb:"5" json:"postal_code"`
+}
+
+// Location holds latitude and longitude information
+type Location struct {
```

Comment:
Refer to google's latlong type https://github.com/googleapis/go-genproto/blob/c506a9f9061087022822e8da603a52fc387115a8/googleapis/type/latlng/latlng.pb.go#L29